pkg/security/probe/serializers.go

```diff
@@ -317,6 +317,13 @@ type SpliceEventSerializer struct {
 	PipeExitFlag  string `json:"pipe_exit_flag" jsonschema_description:"Exit flag of the fd_out pipe passed to the splice syscall"`
 }
 
+// BindEventSerializer serializes a bind event to JSON
+// easyjson:json
+type BindEventSerializer struct {
+	AddrFamily string            `json:"addr_family" jsonschema_description:"Address family"`
```

Comment:
Should we put "family" below the "addr" field like "ip" and "port" ?

Reply:
You're right, it's cleanest. Updated on last commit. Thanks